rdmo/accounts/models.py

```diff
@@ -146,6 +151,21 @@ class Meta:
     def __str__(self):
         return self.user.username
 
+    @property
```

Comment:
Please don't hard code group names. If a group can access something or not should only be determined by the group's permissions.

I don't think we need this property, since this should be covered by model-based-permissions.

Reply:
yes thanks, makes sense, I remove this property